src/doc-utils.js

```diff
@@ -21,6 +21,10 @@ function traverseDoc(doc, onEnter, onExit) {
       if (doc.flatContents) {
         traverseDocRec(doc.flatContents);
       }
+    } else if (doc.type === "group" && doc.expandedStates) {
+      if (shouldTraverseConditionalGroups) {
+        doc.expandedStates.forEach(traverseDocRec);
+      }
```

Comment:
But it traverses into `contents` in a later branch which is the first group of a conditional group